packages/neuron-wallet/src/services/hardware/ledger.ts

```diff
@@ -75,7 +75,7 @@ export default class Ledger extends Hardware {
 
   async signMessage (path: string, messageHex: string) {
     const message = HexUtils.removePrefix(messageHex)
-    const signed = await this.ledgerCKB!.signMessage(path, message, false)
+    const signed = await this.ledgerCKB!.signMessage(path === Address.pathForReceiving(0) ? this.defaultAddress : path, message, false)
```

Comment:
I guess `this.defaultAddress` should be named as `defaultPath`?

Reply:
Yes, the naming is indeed bad.